stdlib/2and3/traceback.pyi

```diff
@@ -36,6 +36,7 @@ if sys.version_info >= (3, 5):
     def extract_stack(f: Optional[FrameType] = ...,
                       limit: Optional[int] = ...) -> StackSummary: ...
     def format_list(extracted_list: List[FrameSummary]) -> List[str]: ...
+    def print_list(extracted_list: List[FrameSummary], file: Optional[IO[str]] = ...) -> None: ...
```

Comment:
1. This should be marked with `# undocumented`.
2. While `StackSummary.from_list()` is also annotated as expecting a list, expecting an `Iterable[FrameSummary]` should suffice.
3. `print()` is annotated as expecting an `IO[str]`. We are trying to move away from using `IO` for argument types. Could you defined a protocol like this and use it for the file argument:
```python
class _Writable(Protocol):
    def write(self, s: str) -> Any:
        pass
```

I will send a separate PR for `print()`.

Reply:
Will do. Thanks for the commends and suggestions!

Reply:
For reference, the print PR is #2848.

Reply:
Looks like all tests have finished but status is still pending...